exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -748,6 +771,22 @@ public ParquetTableMetadataDirs(List<String> directories) {
       return directories;
     }
 
+    /** If directories list contains relative paths, update it to absolute ones
+     * @param baseDir base parent directory
+     */
+    @JsonIgnore public void updateRelativePaths(Path baseDir) {
+      if (!directories.isEmpty()) {
+        // It is enough to check the first path to decide if updating needed
+        if (!new Path(directories.get(0)).isAbsolute()) {
```

Comment:
This is getting a bit silly, converting to/from String and Path. The `directories` list should contain Path elements. In general, we should never work with Path as strings except if we need to serialize as in Jackson.